### ISSUE TITLE
Making this repo into an object detection API

### ISSUE THREAD

**Issue by Abhijit-2592** [yhenon/keras-frcnn#176]
# THIS IS NOT AN ISSUE

Any one interested in making this repo into an object detection api (similar to Tensorflow's object detection API)?
The aim of this API is:
**You can extend almost any keras CNN classifier into an object detection network by plugging your custom network as the fixed feature extractor with minor modifications (passed as arguments).**
Obviously this API will support only Faster-RCNN algorithm (not YOLO and SSD).
This feature will be useful to all keras fans 

I want help in making this API.

If interested comment below with your email-id
Also please share if you have any other similar ideas

Thanks

**Comment by rteja1113** [comment]
There is already tensorflow's object detection API.Check out https://github.com/tensorflow/models/tree/master/object_detection

**Comment by Abhijit-2592** [comment]
Ik that's y I said similar to it  :P

**Comment by 24hours** [comment]
This is an interesting idea. I am assuming you mean implementing it as meta-architecture which described in  tensorflow object detection paper. I could help in writing some of the code. 

I am implementing a GUI tools which allow user to create label and train a network to perform similar task ( basically annotator + deep learning ). The API should come in handy for my project.

**Comment by Abhijit-2592** [comment]
Thanks for your support.
Yeah I meant the meta architecture idea. Your idea of integrating annotation tool in the API is a nice idea. Should be cool

**Comment by yhenon** [comment]
Unfortunately this repo will be hard to make into a proper object detection API due to having many hacks in the code that I had to work around various limitations.

Check out this repo https://github.com/broadinstitute/keras-rcnn where a number of us are working on making a high quality object detection repo for keras

**Comment by Abhijit-2592** [comment]
Thanks @yhenon  for this implementation. I have extended this repo with more additional features. Kindly look at this [repo](https://github.com/Abhijit-2592/Keras_object_detection).
